script/pgsqlms

```diff
@@ -74,6 +74,186 @@ my $ATTRD_PRIV    = "$HA_SBIN_DIR/attrd_updater --private --lifetime reboot";
 # Global vars
 my $nodename;
 my $exit_code = 0;
+# numeric pgsql versions
+my $PGVERNUM;
+my $PGVER_10  = 100000;
+
+# Run a query using psql.
+#
+# This function returns an array with psql return code as first element and
+# the result as second one.
+#
+sub _query {
```

Comment:
I just moved the sub "_query" here, no code modification, you can jump this sub in your review.